From 3a7b636ffda609713b88f1647f0606b037c5ebde Mon Sep 17 00:00:00 2001
From: pobthebuilder <pobthebuilder@pobthebuilder.org>
Date: Sat, 17 Feb 2024 21:46:26 +0000
Subject: [PATCH] Fix hipcc build not to link libstdc++fs.so

---
 amd/hipcc/CMakeLists.txt | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/amd/hipcc/CMakeLists.txt b/amd/hipcc/CMakeLists.txt
index 7ed25cb547d0..6f3182ef618e 100755
--- a/amd/hipcc/CMakeLists.txt
+++ b/amd/hipcc/CMakeLists.txt
@@ -18,9 +18,6 @@ option(BUILD_SHARED_LIBS "Build using shared libraries" ON)
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED True)
 
-set(ADDITIONAL_SHARED_LIBRARIES_TO_LINK
-  libstdc++fs.so)
-
 set(HIPCC_BIN
   hipcc)
 set(HIPCC_SOURCES
-- 
2.49.0

