From 217253017543c3e11f9243fc8c3d826d387a9534 Mon Sep 17 00:00:00 2001
From: pobthebuilder <pobthebuilder@pobthebuilder.org>
Date: Tue, 23 Jan 2024 21:34:22 -0800
Subject: [PATCH] Fix libnuma to use pkg_config

---
 libhsakmt/CMakeLists.txt | 12 ++++--------
 1 file changed, 4 insertions(+), 8 deletions(-)

diff --git a/libhsakmt/CMakeLists.txt b/libhsakmt/CMakeLists.txt
index 8b0ad114..cd5ae88f 100644
--- a/libhsakmt/CMakeLists.txt
+++ b/libhsakmt/CMakeLists.txt
@@ -159,14 +159,9 @@ get_os_info()
 find_package(PkgConfig)
 # Check for libraries required for building
 find_library(LIBC NAMES c REQUIRED)
-find_package(NUMA)
-if(NUMA_FOUND)
-  set(NUMA "${NUMA_LIBRARIES}")
-else()
-  find_library(NUMA NAMES numa REQUIRED)
-endif()
 message(STATUS "LIBC: " ${LIBC})
-message(STATUS "NUMA: " ${NUMA})
+
+pkg_check_modules(NUMA REQUIRED numa)
 
 ## If environment variable DRM_DIR is set, the script
 ## will pick up the corresponding libraries from that path.
@@ -180,9 +175,10 @@ pkg_check_modules(DRM REQUIRED IMPORTED_TARGET libdrm)
 pkg_check_modules(DRM_AMDGPU REQUIRED IMPORTED_TARGET libdrm_amdgpu)
 include_directories(${DRM_AMDGPU_INCLUDE_DIRS})
 include_directories(${DRM_INCLUDE_DIRS})
+include_directories(${NUMA_INCLUDE_DIRS})
 
 target_link_libraries ( ${HSAKMT_TARGET}
-  PRIVATE ${DRM_LDFLAGS} ${DRM_AMDGPU_LDFLAGS} pthread rt ${LIBC} ${NUMA} ${CMAKE_DL_LIBS}
+  PRIVATE ${DRM_LDFLAGS} ${DRM_AMDGPU_LDFLAGS} pthread rt ${LIBC} ${NUMA_LDFLAGS} ${CMAKE_DL_LIBS}
 )
 
 target_compile_options(${HSAKMT_TARGET} PRIVATE ${DRM_CFLAGS} ${HSAKMT_C_FLAGS})
-- 
2.51.0

