From 56bb2d368c3fd86b7ebcac8aef77074d8be1028e Mon Sep 17 00:00:00 2001
From: Segfault <awarnecke002@hotmail.com>
Date: Sun, 4 May 2025 16:29:30 +1000
Subject: [PATCH] Remove use of /etc/os-release

---
 opencl/packaging/CMakeLists.txt | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/opencl/packaging/CMakeLists.txt b/opencl/packaging/CMakeLists.txt
index 1170e57e6..bd4fae7cd 100644
--- a/clr/opencl/packaging/CMakeLists.txt
+++ b/clr/opencl/packaging/CMakeLists.txt
@@ -89,13 +89,6 @@ if(BUILD_ICD)
   set(CPACK_RPM_ASAN_PACKAGE_REQUIRES "comgr-asan, hsa-rocr-asan, rocm-core-asan")
 else()
   set(OCL_ICD_PACKAGE "ocl-icd")
-  if(${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
-    file(READ "/etc/os-release" OS_RELEASE_CONTENT)
-    string(TOLOWER "${OS_RELEASE_CONTENT}" OS_RELEASE_CONTENT_LOWER)
-    if(OS_RELEASE_CONTENT_LOWER MATCHES "suse" OR OS_RELEASE_CONTENT_LOWER MATCHES "sles")
-      set(OCL_ICD_PACKAGE "ocl-icd-devel")
-    endif()
-  endif()
   set(CPACK_RPM_BINARY_PACKAGE_REQUIRES "comgr, hsa-rocr, rocm-core, ${OCL_ICD_PACKAGE}")
   set(CPACK_RPM_ASAN_PACKAGE_REQUIRES "comgr-asan, hsa-rocr-asan, rocm-core-asan")
 endif()
-- 
2.49.0

