From c588f583219542ae96709a08825faaa1360b8a2b Mon Sep 17 00:00:00 2001
From: pobthebuilder <pobthebuilder@pobthebuilder.org>
Date: Sun, 14 May 2023 20:12:30 -0700
Subject: [PATCH] Find drm and drm_amdgpu paths

---
 CMakeLists.txt | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index cbcf9d0..02211f9 100755
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -57,7 +57,10 @@ include(utils)
 # Process input variables
 #
 
+find_package(PkgConfig REQUIRED)
 find_package(hsa-runtime64 1.0 REQUIRED )
+pkg_check_modules(drm REQUIRED IMPORTED_TARGET libdrm)
+pkg_check_modules(drm_amdgpu REQUIRED IMPORTED_TARGET libdrm_amdgpu)
 
 string(TOLOWER "${ROCRTST_BLD_TYPE}" tmp)
 if("${tmp}" STREQUAL release)
@@ -166,7 +169,7 @@ configure_file(rocm_agent_enumerator rocm_agent_enumerator COPYONLY)
 ###########################
 aux_source_directory(${CMAKE_CURRENT_SOURCE_DIR} ROCMINFO_SOURCES)
 add_executable(${ROCMINFO_EXE} ${ROCMINFO_SOURCES})
-target_link_libraries(${ROCMINFO_EXE} hsa-runtime64::hsa-runtime64)
+target_link_libraries(${ROCMINFO_EXE} hsa-runtime64::hsa-runtime64 PkgConfig::drm PkgConfig::drm_amdgpu)
 
 target_compile_options(${ROCMINFO_EXE} PRIVATE ${ROCMINFO_CXX_FLAGS})
 
-- 
2.40.0

