From 40120cbcfb703f179e83d141875c793698564fcc Mon Sep 17 00:00:00 2001
From: pobthebuilder <pobthebuilder@pobthebuilder.org>
Date: Tue, 23 Jan 2024 21:34:22 -0800
Subject: [PATCH] Fix libnuma to use pkg_config

---
 CMakeLists.txt | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7e78bfb..341a622 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -162,7 +162,8 @@ find_package(PkgConfig)
 find_library(LIBC NAMES c REQUIRED)
 find_library(NUMA NAMES numa REQUIRED)
 message(STATUS "LIBC:" ${LIBC})
-message(STATUS "NUMA:" ${NUMA})
+
+pkg_check_modules(NUMA REQUIRED numa)
 
 ## If environment variable DRM_DIR is set, the script
 ## will pick up the corresponding libraries from that path.
@@ -181,9 +182,10 @@ else()
     include_directories(${DRM_AMDGPU_INCLUDE_DIRS})
 endif()
 include_directories(${DRM_INCLUDE_DIRS})
+include_directories(${NUMA_INCLUDE_DIRS})
 
 target_link_libraries ( ${HSAKMT_TARGET}
-  PRIVATE ${DRM_LDFLAGS} ${DRM_AMDGPU_LDFLAGS} pthread rt c numa
+  PRIVATE ${DRM_LDFLAGS} ${DRM_AMDGPU_LDFLAGS} pthread rt ${LIBC} ${NUMA_LDFLAGS}
 )
 
 target_compile_options(${HSAKMT_TARGET} PRIVATE ${DRM_CFLAGS} ${HSAKMT_C_FLAGS})
-- 
2.41.0

